src/hooks/useThemeProps/index.ts

```diff
@@ -0,0 +1,13 @@
+import {useContext, createContext} from 'react';
+import * as Modifiers from '../../commons/modifiers';
+import {ThemeManager} from 'style';
+
+const EmptyContext = createContext({});
+
+const useThemeProps = (props: any, componentName: string) => {
+  const themeContext = ThemeManager.getThemeContext();
```

Comment:
Wouldn't it be better to return empty theme context inside `.getThemeContext` or is that not an option?